### PR TITLE
feat: up to date case data class (CMC-NA)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

N/A

### Change description ###

Up to date case data class

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by dharmendrak — DISMISSED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/ucmc/model/SolicitorReferences.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SolicitorReferences {
+    private final String solicitorReference;
```
Comment (sabahirfan) [inline src/main/java/uk/gov/hmcts/reform/ucmc/model/SolicitorReferences.java]: I think we have renamed this to `claimantReference`.

Comment (swalker125) [reply src/main/java/uk/gov/hmcts/reform/ucmc/model/SolicitorReferences.java]: yeah I've made the mistake of building this off of 353 🤦

**Review by sabahirfan — DISMISSED** [review]
LGTM

**Pull request closed without merging**